test/unit/YawPitchControl/RotationPanInput.spec.js

```diff
@@ -0,0 +1,93 @@
+import {window} from "../../../src/YawPitchControl/browser";
+import RotationPanInput from "../../../src/YawPitchControl/input/RotationPanInput";
+import RotationPanInputInjector from "inject-loader!../../../src/YawPitchControl/input/RotationPanInput";
+
+describe("RotationPanInput", function() {
+	describe("#constructor", function() {
+		beforeEach(() => {
+			this.el = sandbox();
+		});
+
+		afterEach(() => {
+			this.el = sandbox();
+
+			this.inst.destroy();
+		});
+
+		it("Instance", () => {
+			// Given
+			// When
+			this.inst = new RotationPanInput(this.el);
+
+			// Then
+			expect(this.inst).to.be.exist;
+		});
+
+		it("Instance with useRotation: true", () => {
+			// Given
+			// When
+			this.inst = new RotationPanInput(sandbox(), {useRotation: true});
```

Comment:
You intended `this.el` than `sandbox()`?

Reply:
OK